### ISSUE TITLE
Default baud rate should take OS into account

### ISSUE THREAD

**Issue by kenr** [NordicSemiconductor/pc-ble-driver-js#142]
The default baud rate when baud rate is not specified is 115200.
This baud rate is only relevant for macOS.

The open call should take the OS into account when setting a default baud rate:
https://github.com/NordicSemiconductor/pc-ble-driver-js/blob/6194a45c08d65769ed7bb1052fc6d77b1acf3bbb/api/adapter.js#L380

**Comment by jrobeson** [comment]
the next version (i think) of pc-ble-driver supports larger baud rates on Mac OS

**Comment by bihanssen** [comment]
True, we could standardize on 1Mbps now that it is supported on all platforms.
